blockstore/splitstore/splitstore.go

```diff
@@ -529,6 +568,7 @@ func (s *SplitStore) Start(chain ChainAccessor) error {
 func (s *SplitStore) Close() error {
 	atomic.StoreInt32(&s.closing, 1)
 
+	// I'm _pretty_ sure we can just use a lock here now. Doesn't even need to be a rwmutex.
```

Comment:
I was using a lock way back when the code was first written but raul was adamant that we ought to use an atomic here as it is exactly the pattern that calls for it.